src/Transport/GrpcFallbackTransport.php

```diff
@@ -90,7 +95,8 @@ public static function build(string $apiEndpoint, array $config = [])
         ];
         list($baseUri, $port) = self::normalizeServiceAddress($apiEndpoint);
         $httpHandler = $config['httpHandler'] ?: self::buildHttpHandlerAsync();
-        $transport = new GrpcFallbackTransport("$baseUri:$port", $httpHandler);
+        $logger = isset($config['logger']) ? $config['logger'] : null;
```

Comment:
Use more succinct ternary/null-coalescing:
```suggestion
        $logger = $config['logger'] ?? null;
```
... or do it inline, as I suggest below

Reply:
If `logger` is not set, that would raise a warning, wouldn't it? Wouldn't crash, but it would throw a warning.